community/kernel/src/main/java/org/neo4j/kernel/api/cursor/LabelValue.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.api.cursor;
+
+import org.neo4j.function.IntSupplier;
+
+/**
+ * Represents a label value from {@link LabelCursor}.
+ */
+public interface LabelValue
```

Comment:
Suggest -1 to exposing it as a supplier unless there's a need for that, otherwise we're implementing code with no user driving it.